common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt

```diff
@@ -279,4 +271,144 @@ d_burning_blade = {
 		color1 = "red"
 		color2 = "orange"
 	}
+}
+
+### Dwarfs
+e_khaz_modan = {
+	pattern = "pattern_solid.dds"
+	color1 = rgb { 34 55 7 }
+	colored_emblem = {
+		texture = "ce_khaz_modan.dds"
+		color1 = rgb { 88 83 96 }
+		color3 = rgb { 147 101 37 }
+		color2 = rgb { 89 191 34 }
+		instance = { position = { 0.5 0.5 } scale = { 0.9 0.9 }  }
+	}
+}
+
+k_ironforge = {
+	pattern = "pattern_solid.dds"
+	color1 = rgb { 173 70 1 }
+	colored_emblem = {
+		texture = "ce_ironforge.dds"
+		color1 = rgb { 195 189 129 }
+		color2 = rgb { 224 188 4 }
+		color3 = rgb { 207 134 8 }
+		instance = { position = { 0.5 0.5 } scale = { 0.9 0.9 }  }
+	}
+}
+
+k_blackrock = {
+	pattern = "pattern_solid.dds"
+	color1 = "brown_darkiron"
+	colored_emblem = {
+		texture = "ce_shadowforge.dds"
+		color1 = rgb { 54 48 41 }
+		color2 = rgb { 156 122 93 }
+		color3 = rgb { 107 71 39 }
+	}
+}
+
+k_wildhammer = {
+	pattern = "pattern_solid.dds"
+	color1 = rgb { 34 55 7 }
+	colored_emblem = {
+		texture = "ce_grim_batol.dds"
+		color1 = rgb { 147 101 37 }
+		color3 = rgb { 130 82 23 }
+	}
+}
+
+k_wetlands = {
```

Comment:
I would use brighter colors here, it's too dark.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/121759899-0147f700-cb39-11eb-91a8-f21098de8be9.png)

</details>